indico/modules/events/abstracts/__init__.py

```diff
@@ -0,0 +1,15 @@
+# This file is part of Indico.
```

Comment:
i usually keep empty files completely empty and just add the header when adding actual content